Sources/SwiftKueryPostgreSQL/PostgreSQLConnection.swift

```diff
@@ -491,4 +504,28 @@ public class PostgreSQLConnection: Connection {
         PQclear(result)
         onCompletion(.successNoData)
     }
+
+    private func buildQuery(_ query: Query) throws  -> String {
+      var postgresQuery = try query.build(queryBuilder: queryBuilder)
+      if (query is Insert) {
+
+        guard let insertQuery = query as? Insert,
+              let returnIDFlag = insertQuery.returnID
+        else {
+          throw QueryError.downcastError("Error downcasting Query to Insert")
+        }
```

Comment:
Why not just `if let insertQuery = query as? Insert {`? Is there something I'm missing? I don't think there shouldn't ever be a downcast error here.

Reply:
Done